app/src/main/java/appricottsoftware/clarity/HomeActivity.java

```diff
@@ -202,14 +206,15 @@ public void onBackPressed() {
             } else if (!(fragment instanceof HomeFragment)) {
                 returnToHomeFragment();
             } else {
-                // Exit app
+                // Exits app
                 finish();
+
+                // Statement below used to be here. Keeping just in case.
+                // moveTaskToBack(true);
```

Comment:
Get rid of this dead code if not needed.

Reply:
Not sure if it's needed or not yet. It used to be in Karen's old code, so I'm not sure what the implications are of removing it. As of right now everything works fine without it though.. Maybe we should get @kkong006's input